ui/src/components/table/CustomTableStyle.jsx

```diff
@@ -6,6 +6,9 @@ import WaitSpinner from '@splunk/react-ui/WaitSpinner';
 export const ActionButtonComponent = styled(Button)`
     margin: 0px 1px;
     border: none;
+    display: flex !important;
```

Comment:
```suggestion
    display: flex;
```

Reply:
Please use !important as last resort

Reply:
Yes, I have checked all the scenarios but when I gave the styling without !important, then it did not override the style.

Reply:
`inline` you set for the Button overrides your `display: flex`. Instead of setting important, remove the obstacle - `inline` prop